### ISSUE TITLE
Update homepage for Advocacy.org

### ISSUE THREAD

**Issue by cassiemc** [MozillaFoundation/movement-building#73]

**Comment by cassiemc** [comment]
Potential new wireframe from @kristinashu, we'll be discussing potential to implement this this afternoon. https://redpen.io/dwc1eb0d59700704ba
cc @lovegushwa @HPJohnson @brettgaylor @Melechuga @ScottDowne @valianttry

**Comment by valianttry** [comment]
hi everyone. The advocacy.mozilla.org domain pre-dates our recent organizational changes, so the concept is a bit outdated (e.g. Open Web Fellows + campaigns). I thought the focus of the redesign was for advocacy.mozilla.org to become primarily (solely?) the home for Open Web Fellows and de-emphasize campaigns. Our campaigns will have their own landing pages and content and don't need to have prominent placement at advocacy.mozilla.org. The thinking behind this is that "advocacy" is integrated as part of Mozilla's work and not a standalone project with its own domain.

**Comment by cassiemc** [comment]
Interesting! Thanks for that insight @valianttry. Would love to hear @davidascher's thoughts too. Seems like @HPaulJohnson @lovegushwa and @Melechuga were onboard with building out a placeholder / home for past and ongoing campaigns, as per the redpen, which seemed inline with our rationale for keeping the web fellows stuff on the advocacy url ("our open web fellows program is one of the ways in which we express our advocacy work at mozilla"). 

But I'm definitely open to whatever the best solution is programatically. A fellows homepage is certainly within the realm of possibility at this point. Also – am sending out an email shortly re. the group discussion we had an hour ago.

**Comment by davidascher** [comment]
@valianttry: Where do we expect people who are interested in becoming
grassroots organizers, find out how we're working, or otherwise get
involved in advocacy should follow along?  IOW, what's the advocacy
equivalent of fundraising.mozilla.org?

**Comment by valianttry** [comment]
I chatted with @HPaulJohnson today, and I understand the campaign content will be minimal on advocacy.mozilla.org. We won't be driving traffic to it, I understand the concept is that it's basically a "repository"of our campaigns. 

@davidascher so far @shaghdoosti and I have blogged on our personal blogs, but it's a great question. Right now advocacy.mozilla.org isn't set up as a blog so I'm not sure if that's in scope for updates as planned currently. A blog could live at /strategy maybe? It's a good idea.

**Comment by cassiemc** [comment]
@valianttry Yes that's the idea. Also, we are going to figure out a blogging solution (lots of people in the same position as you) via a larger templatization plan – https://github.com/MozillaFoundation/plan/issues/644, so if there's no rush to move on a blog we could make what you need via that larger task. The plan with @brettgaylor and @Melechuga for now is to link to the Policy blog, which is what last years' open web fellows did. We are looking into whether there can be a specific tag added to filter open web fellows specific posts.

**Comment by ScottDowne** [comment]
Does anyone know if we want to now or in the future localise this?

**Comment by cassiemc** [comment]
@ScottDowne I don't know the answer to that but worth pinging @valianttry @brettgaylor @HPaulJohnson on.

I set up a meeting with @HPaulJohnson, @KevZawacki and @kristinashu and I to go over content for this landing page tomorrow. Goal is to finalize design for this page this week, and have a healthy headstart on content as well.

**Comment by brettgaylor** [comment]
I don't see localizing this as a priority.

**Comment by cassiemc** [comment]
Thanks @brettgaylor! Just met with a little group of folks and the plan is:
- @lovegushwa / @HPaulJohnson can update the encryption team at today's stand about the plan for this advocacy homepage. Let us know if there are any flags or if anything seems of a concern, and if other people want/need to be involved.
- @kristinashu to adapt wireframe to add a CTA to sign up to mailing list; some of the content here will likely adjust (ie. instead of naming the encryption campaign for this page, likely just a headline and/or CTA; and instead of having 'past campaign's they might just be links to blog posts or older initiatives to help describe some history) but overall structure looks good.
- @KevZawacki & @HPaulJohnson to write content for homepage, we're aiming for end of week or early next week. Content types are: encryption headline & cta, email opt-in, advocacy program overview (1-3 points if possible, otherwise we can adapt design to a longer paragraph), and bullet points for the 'past campaigns' section
- @kristinashu @lovegushwa and @natalieworth to collaborate on further refining the UI, but seems close (we're agreed that we are designing a first run; can iterate on this later so won't introduce too many new styles/elements that dictate direction of the advocacy brand)

We also reviewed the encryption landing page wireframes and all looks really good from a streamlining point of view – great job @natalieworth and @lovegushwa and @kristinashu working together on this. @ScottDowne shouldn't have a problem building all these together as long as we can make sure the UI meets all the needs.

Go team go!

**Comment by kristinashu** [comment]
Made some slight updates to the home page wireframe: https://redpen.io/p/ds7226d960fbdeed27

**Comment by lovegushwa** [comment]
Hey folks, Brett is now on PTO for a few weeks. We looked at the
wire-frames today in our stand-up and no one voiced any concerns.
Let's move ahead with this, @HPaulJohnson and @KevZawacki need to suggest
any final copy changes / what prior advocacy content (blog post's etc.) to
launch with.

**Comment by cassiemc** [comment]
I'm going to close this bug in favour of a clean one for building this page UI, once it is approved. New issue is #80
